eng/pipelines/release-cli.yml

```diff
@@ -986,12 +986,42 @@ stages:
                     filePath: eng/scripts/Set-CliVersionVariable.ps1
                   displayName: Set CLI_VERSION
 
+                - pwsh: |
+                    # Initial upload locations
+                    $publishUploadLocations = 'release/$(CLI_VERSION);release/latest'
+
+                    $isPublishingGa = eng/scripts/Test-ShouldReleasePackageVersion.ps1 `
+                      -CliVersion '$(CLI_VERSION)'
+
+                    if ($isPublishingGa) {
+                      $publishUploadLocations += ";release/stable"
+                    }
+
+                    Write-Host "Setting StorageUploadLocations to $publishUploadLocations"
+                    Write-Host "###vso[task.setvariable variable=StorageUploadLocations]$publishUploadLocations"
+                  displayName: Set StorageUploadLocations
+
+                - pwsh: |
+                    # Initial tag
+                    $dockerTags = '$(CLI_VERSION)'
+
+                    $isPublishingGa = eng/scripts/Test-ShouldReleasePackageVersion.ps1 `
+                      -CliVersion '$(CLI_VERSION)'
+
+                    if ($isPublishingGa) {
+                      $dockerTags += ";latest"
```

Comment:
I wonder if we are going to want `stable` and `latest` tags in the future that correspond to the `stable` and `latest` channels we have for the installer script?  Maybe something reasonable to do later, if there's demand (my hope is that long term we can get out of the business of producing this giant docker image in favor of giving folks an easy way to install azd into their container they are building (either using the install script, or by adding a dev container feature thing for dev container scenarios)).

Reply:
In the case of Docker, my thinking is that `latest` is what most people will use by default. In that case, as also is the case with install scripts and package managers (e.g. choco/winget), the customer is going to be surprised if they suddenly are installing bits that are of a `beta` version. 

So we keep `latest` in docker to only stable releases now. We could set up a beta or daily channel if it's requested, however, the greatest benefit probably comes from the customer easily installing the desired version in their own container build process. We support that today with the install scripts.